app/src/layouts/organizationsControl/organizationsControl.jsx

```diff
@@ -42,15 +42,21 @@ export const OrganizationsControl = ({
       <div>
         <div
           className={cx('organization-btn-wrapper', {
-            'instance-level': isExtendedNav,
+            'instance-level': !isExtendedNav,
```

Comment:
Let's rename the class to `not-extended` or `plan` or smth like that.

Reply:
Done